qadence/backends/pulser/backend.py

```diff
@@ -48,20 +44,19 @@ def _convert_init_state(state: Tensor) -> np.ndarray:
     return np.flip(state.cpu().squeeze().numpy())
 
 
-def create_register(register: Register, spacing: float = DEFAULT_SPACING) -> PulserRegister:
+def create_register(register: Register) -> PulserRegister:
```

Comment:
I think this function is used only once, maybe we can even get rid of it now?

Reply:
I think it's ok here just for the sake of organization (and potential reuse) but I simplified the docstring